kafka-client/src/test/java/dev/responsive/kafka/integration/ResponsiveSessionStoreIntegrationTest.java

```diff
@@ -180,31 +179,13 @@ public void shouldComputeSessionAggregate() throws Exception {
       startAppAndAwaitRunning(Duration.ofSeconds(15), kafkaStreams);
       pipeRecords(producer, inputTopic(), inputEvents);
 
+      final boolean awaited = outputLatch.await(20_000, TimeUnit.MILLISECONDS);
       assertThat(
-          outputLatch.await(20_000, TimeUnit.MILLISECONDS),
-          Matchers.equalTo(true)
-      );
-
-      assertThat(actualPeeks, Matchers.hasSize(expectedPeeks.size()));
-      for (var i = 0; i < actualPeeks.size(); i++) {
-        var actual = actualPeeks.get(i);
-        var expected = expectedPeeks.get(i);
-        assertThat(actual.key, Matchers.equalTo(expected.key));
-        assertThat(actual.value, Matchers.equalTo(expected.value));
-      }
-    }
-
-    properties.put(ResponsiveConfig.STORE_FLUSH_RECORDS_TRIGGER_CONFIG, 0);
-    producer = new KafkaProducer<>(properties);
-    try (
-        final ResponsiveKafkaStreams kafkaStreams =
-            new ResponsiveKafkaStreams(builder.build(), properties);
-    ) {
-      startAppAndAwaitRunning(Duration.ofSeconds(15), kafkaStreams);
-      pipeRecords(producer, inputTopic(), inputEvents);
-
-      assertThat(
-          outputLatch.await(20_000, TimeUnit.MILLISECONDS),
+          String.format(
+              "The application did not receive the expected number of peeks: %d / %d",
```

Comment:
let's just log all the peeks instead of just the number - it's not that many events :) that will tell us whether we're missing something in the middle vs something in the end